tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -295,17 +296,24 @@ private static string GetRegistryName(string repo, string tag)
 
                 if (repoInfo["images"] != null)
                 {
-                    imageExistsInStaging = repoInfo["images"]
+                    isUsingCustomRegistry = repoInfo["images"]
                         .Cast<JProperty>()
                         .Any(imageInfo => imageInfo.Value["simpleTags"].Any(imageTag => imageTag.ToString() == tag));
                 }
                 else
                 {
-                    imageExistsInStaging = false;
+                    isUsingCustomRegistry = false;
                 }
             }
 
-            return imageExistsInStaging ? $"{Registry}/{RepoPrefix}" : "mcr.microsoft.com/";
+            if (isUsingCustomRegistry)
```

Comment:
After looking at the implementation more closely, I'm not certain these asserts are valid.  The registry gets defaulted to mcr.microsoft.com if the env var is not specified.  Also it is valid to have an empty Prefix.  I think the original pattern was better.